### PR TITLE
I changed every gitter reference to matrix

### LINKED ISSUE #1084: Replace references to the Gitter room with matrix
The matrix channel appears to have been more active than the gitter channel recently, should we replace links and stuff everywhere with the matrix channel, or link to both?

### PR DESCRIPTION
**Pull request by alexmarginean16**
# Description

I tried to change all references of gitter to the matrix channel. [this](https://github.com/opsdroid/opsdroid/search?q=%22gitter.im%2Fopsdroid%22&unscoped_q=%22gitter.im%2Fopsdroid%22) helped me a lot.

Fixes #1084 

## Status
**READY**

## Type of change

- Documentation (fix or adds documentation)

# How Has This Been Tested?

I changed the links everywhere where it was found by the search engine.

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation (if applicable)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
+<<<<<<< HEAD
```
Comment (FabioRosado) [inline README.md]: This shows the start of the conflict that git has found, if you try to see this file in rich diff, you will see that the header of the readme is duplicated. You will need to delete one of them.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+[![codecov](https://img.shields.io/codecov/c/github/opsdroid/opsdroid.svg)](https://codecov.io/gh/opsdroid/opsdroid)
+[![BCH compliance](https://bettercodehub.com/edge/badge/opsdroid/opsdroid?branch=master)](https://bettercodehub.com/)
+[![Updates](https://pyup.io/repos/github/opsdroid/opsdroid/shield.svg)](https://pyup.io/repos/github/opsdroid/opsdroid/)
+[![Docker Build](https://img.shields.io/docker/build/opsdroid/opsdroid.svg)](https://hub.docker.com/r/opsdroid/opsdroid/builds/)
+[![Docker Image](https://img.shields.io/microbadger/image-size/opsdroid/opsdroid.svg)](https://hub.docker.com/r/opsdroid/opsdroid/)
+[![Docker Layers](https://img.shields.io/microbadger/layers/opsdroid/opsdroid.svg)](https://microbadger.com/#/images/opsdroid/opsdroid)
+[![Documentation Status](https://img.shields.io/readthedocs/opsdroid/latest.svg)](http://opsdroid.readthedocs.io/en/stable/?badge=stable)
+[![Matrix Chat](https://img.shields.io/matrix/opsdroid-general:matrix.org.svg?logo=matrix)](https://riot.im/app/#/room/#opsdroid-general:matrix.org)
+[![Badge](https://img.shields.io/badge/gitter-join%20chat-4fb896.svg)](https://gitter.im/opsdroid)
```
Comment (FabioRosado) [inline README.md]: This is the gitter badge, since we are not using gitter anymore this should be removed from the list.

File: README.md [hunk 3]
```diff
@@ -41,6 +64,7 @@ An open source chat bot framework written in Python. It is designed to be extend
 
 
 This application is designed to take messages from chat services and execute Python functions (skills) based on their contents. Those functions can be anything you like, from simple conversational responses to running complex tasks. The true power of this project is to act as a glue library to bring the multitude of natural language APIs, chat services and third party APIs together.
+>>>>>>> upstream/master
```
Comment (FabioRosado) [inline README.md]: This is the end of the git conflict

File: docs/contributing.md [hunk 4]
```diff
@@ -9,8 +9,13 @@ Things you can help with:
  - [Issues](#issues)
      - [Squashing bugs](#quick-links)
      - [Enhance opsdroid](#quick-links)
+<<<<<<< HEAD
```
Comment (FabioRosado) [inline docs/contributing.md]: This is showing another conflict, check the difference between the two and go with the right one

File: docs/contributing.md [hunk 5]
```diff
@@ -178,8 +183,13 @@ Then you can translate it in `locale/eo/LC_MESSAGES/opsdroid.po`, then compile i
 Opsdroid is an open source chatbot framework. It is designed to be extendable, scalable and simple. It comes with a few official modules that can be found in the [Opsdroid  GitHub account](https://github.com/opsdroid).
 
  If you love a particular platform and wish to use opsdroid with it or if you want opsdroid to interact with something in a certain way you can create your own modules and extend the functionality of opsdroid.
+<<<<<<< HEAD
```
Comment (FabioRosado) [inline docs/contributing.md]: This is the start of the conflict. You should delete the one that is outdated and go with the latest one.

**Comment by alexmarginean16** [comment]
Hey, I tried to solve every conflict. Please let me know if there is any other problem

**Pull request merged**